src/demo.json

```diff
@@ -2,16 +2,6 @@
   "name": "Normal Distribution Calculator",
   "qset": {
     "version": 1,
-    "data": {
-      "items": [
-        {
-          "materiaType": "question",
-          "id": null,
-          "type": "MC",
-          "questions": [],
-          "answers": []
-        }
-      ]
-    }
+    "data": {}
```

Comment:
Widget instances won't save qsets if the `data` property is empty - something has to be in here to avoid problems down the line, at least for now.